src/content/releases/_index.en.md

```diff
@@ -1,10 +1,27 @@
 +++
-date = 2016-04-09T16:50:16+02:00
+date = 2020-05-04T16:50:16+02:00
 title = "Releases"
 pre = "<b>4. </b>"
 weight = 15
 +++
 
+## v0.3.0 Lighthouse service discover without kubefed
+
+> This release is focused on removing the kubefed dependency from Lighthouse, the user experience
+> and adding experimental wireguard support
+
+* Lighthouse does not depend anymore on kubefed and all metadata exchange is handled
+  over the broker as MultiClusterService CRs
+* Experimental *wireguard* support has been added as a Cable driver.
+* Submariner reports the active and passive gateways as a gateway.submariner.io CRD
+* The submariner operator reports detailed status on the deployment.
+* The *gateway redundancy/failover* tests are now enabled and stable on CI
+* *Globalnet hostNetwork* to remote globalIP is supported now
```

Comment:
```suggestion
* *Globalnet hostNetwork* to remote globalIP is now supported.
```

Must admit I don't fully understand this one. :sweat_smile: Would it be possible to add a bit more background/why?

Reply:
```suggestion
* *Globalnet hostNetwork* to remote globalIP is now supported. Previously, when a pod used hostNetworking it was unable to connect to a remote service via globalIP.
```